src/app/helptext/system/certificates.ts

```diff
@@ -206,19 +207,22 @@ matches your certificate usage scenario.'),
     basic_constraints: {
       config: {
         placeholder: T('Basic Constraints Config'),
-        tooltip: T('The basic constraints extension identifies whether the \
- subject of the certificate is a CA and the maximum depth of valid \
- certification paths that include this certificate. <br> \
- See <a href="https://www.ietf.org/rfc/rfc3280.txt">RFC 3280, section 4.2.1.10</a> \
- for more information.'),
+        tooltip: T('Specify whether the certificate may be used for a Certificate Authority,\
```

Comment:
I recommend:

`Specify whether to use the certificate for a Certificate Authority and whether this extension is critical. Clients must recognize critical extensions to prevent rejection. Web certificates typically require you to disable CA and enable Critical Extension.`